### PR TITLE
Tasks 11-20

### PR DESCRIPTION
**Pull request by KamilWasilonek**

### CODE REVIEW

File: tasks/10_remove-whitespaces/remove-whitespaces.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export function trimWhiteSpaces(textToTrim) {
+  let text = textToTrim.split(' ');
```
Comment (tkachukivan) [inline tasks/10_remove-whitespaces/remove-whitespaces.js]: you may have conflict with previous pull request as changes the same file

File: tasks/17_truncate-string/truncate-string.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export function truncateString(str, maxLength) {
+  if (str.length > maxLength) {
+    return str.slice(0, maxLength <= 3 ? maxLength : maxLength -3) + '...';
```
Comment (tkachukivan) [inline tasks/17_truncate-string/truncate-string.js:3]: 👍 
but would be better to move `maxLength <= 3 ? maxLength : maxLength -3` to some variable and than pass this variable to slice function

File: tasks/11_find-longest-word/find-longest-word.spec.js [hunk 3]
```diff
@@ -1,27 +1,27 @@
-// import { findLongestWord } from './find-longest-word';
+import {findLongestWord} from './find-longest-word';
 
-xdescribe('find-longest-word', () => {
-    test('findLongestWord("The quick brown fox jumped over the lazy dog") should return a number', () => {
-        expect(typeof findLongestWord("The quick brown fox jumped over the lazy dog")).toBe('number');
```
Comment (tkachukivan) [inline tasks/11_find-longest-word/find-longest-word.spec.js:5]: check if you turn on formatting on save :) you don't need to change something except the first lines in tests files

File: tasks/1_golf-score/golf-score.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  PAR: `Par`,
+  BOGEY: `Bogey`,
+  DOUBLE_BOGEY: `Double Bogey`,
+  GO_HOME: `Go Home`,
+};
+
+export function golfScore(par, strokes) {
+  if (strokes == 1) return GOLF_SCORES.HOLE_IN_ONE;
+  else if (strokes <= par - 2) return GOLF_SCORES.EAGLE;
+  else if (strokes == par - 1) return GOLF_SCORES.BIRDIE;
```
Comment (tkachukivan) [inline tasks/1_golf-score/golf-score.js]: it may produce conflict

**Review by tkachukivan — APPROVED** [review]
please fix that this pull not include changes for 1-10 tasks

File: tasks/1_golf-score/golf-score.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+  else if (strokes == par) return GOLF_SCORES.PAR;
+  else if (strokes == par + 1) return GOLF_SCORES.BOGEY;
+  else if (strokes == par + 2) return GOLF_SCORES.DOUBLE_BOGEY;
```
Comment (mikhail-ev) [inline tasks/1_golf-score/golf-score.js]: you should use triple equals (===)
https://codeburst.io/javascript-double-equals-vs-triple-equals-61d4ce5a121a

**Review by mikhail-ev — APPROVED** [review]

**Pull request merged**